code/espurna/sensors/PMSX003Sensor.h

```diff
@@ -287,6 +302,12 @@ class PMSX003Sensor : public BaseSensor, PMSX003 {
                     _slot_values[0] = data[3];
                     _slot_values[1] = data[4];
                     _slot_values[2] = data[5];
+                    _slot_values[3] = data[6] * 10; // convert particles per 0.1L of air to 1L
```

Comment:
According to the data sheet:

![screen shot 2018-08-09 at 01 54 17](https://user-images.githubusercontent.com/288709/43871982-2b5ad4c6-9b77-11e8-9b03-a2b184b57f37.png)

This was my interpretation (read value * 10 for number of particles in 1L)

Reply:
Yes, I understand the same, but you have defined the units per square meter of air when it should me per liter, so `magnitude_particles_m3` should be `magnitude_particles_l`